packages/liquidator/src/liquidator.js

```diff
@@ -213,6 +213,20 @@ class Liquidator {
       at: "Liquidator",
       message: "Checking for liquidatable positions and preforming liquidations"
     });
+    if (
```

Comment:
As @mrice32 mentioned I think this would be cleaner in the BalanceMonitor. Wdyt ?

Reply:
hmm. I'm not too sure about that. This is a _liquidator_ specific check and is independent of the monitor bot. If, say, I'm only running a liquidator how would placing this in the `BalanceMonitor` detect this invalid config if I'm not running a monitor?

Another option could be to put it within the liquidator's `index.js` file but it was cleaner to put it here after trying both options.

WDYT?